modules/affile/affile-grammar.ym

```diff
@@ -194,6 +198,19 @@ source_wildcard_option
 	| source_affile_option
 	;
 
+source_legacy_wildcard_option
+	: KW_FORCE_DIRECTORY_POLLING '(' yesno ')'
+	  {
+	    CHECK_ERROR(last_legacy_wildcard_src_driver, @1, "Invalid option, force-directory-polling() can only be used in legacy wildcard file sources");
```

Comment:
I think this is not 'invalid' but 'deprecated' option... (maybe I'm wrong)

Reply:
This is displayed only when one uses these options with a not-legacy wildcard file source (with a regular file source).

Should we allow these options in regular cases with a deprecation warning?

Reply:
I see, okay... no, of course we should not.